package.json

```diff
@@ -2,7 +2,7 @@
   "author": "David Momenso",
   "name": "node-dht-sensor",
   "description": "Reads data from DHT sensors on Raspberry Pi",
-  "version": "0.2.1",
+  "version": "0.3.0",
```

Comment:
Probably this should have been done in a different commit

Reply:
Yes, no need to bump version in the pull-request.